INTEGRATIONS.md

```diff
@@ -1,11 +1,37 @@
-### Integrations
+# Integrations
```

Comment:
Can we split out the header-level and newline changes into their own commit? That way, it's obvious which changes apply to the cleanup, and which apply to the feature.